server/sonar-web/src/main/js/apps/coding-rules/templates/coding-rules-workspace-header.hbs

```diff
@@ -2,14 +2,28 @@
   {{#if state.rule}}
     <a class="js-back">{{t 'coding_rules.return_to_list'}}</a>
   {{else}}
-    &nbsp;
+    {{#if canBulkChange}}
+      <button class="js-bulk-change">{{t 'bulk_change'}}</button>
+    {{/if}}
+    <button class="js-new-search" id="coding-rules-new-search">{{t 'coding_rules.new_search'}}</button>
```

Comment:
I don't find very explicit that new search button, at first I thought I could click on it instead of the "Enter" button to validate my search. Maybe we should rename it "clear search".